client-ui/src/app/effects/execution-events.effects.ts

```diff
@@ -48,7 +48,11 @@ export class ExecutionEventsEffects {
     public getPageableExecutionEvents$: Observable<Action> = this.actions$.pipe(
         ofType(ExecutionEventsAction.ExecutionEventsActionTypes.EXECUTION_EVENTS_GET),
         switchMap((action: any) => this.executionEventControllerService.executionEventUsingGET(action.payload)),
-        map((res: PageableExecutionEvent) => new ExecutionEventsAction.GetSuccess(res))
+        map((res: PageableExecutionEvent) => new ExecutionEventsAction.GetSuccess(res)),
+        catchError(err => {
+            this.store.dispatch(new ErrorActions.ServiceErrorGet(handleError(err)))
+            return empty()
+        })
```

Comment:
This error handling fragment is here several times, would it make sense to refactor it to function?

Reply:
Good point, it should be possible by creating a custom rxjs operator, I will try to add it but I need to read a bit about it first.